src/inverseRobot.js

```diff
@@ -25,6 +25,23 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const output = robot;
+  const values = Object.values(output);
+  const result = {};
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = 0; j < values.length; j++) {
+      if (values[i] === values[j] && i !== j) {
+        return null;
+      }
+    }
+  }
```

Comment:
you could replace this whole code with a neat `result.hasOwnProperty(robot[key])` validation and returning null inside `for` loop

Reply:
but i need to check if the value exist not the key